Update for ModOrganizer 2.5.3

Exit-9B opened the pull request:
Closes #14.

Exit-9B committed:
Revert "Temporarily bump down minimum CMake version"

This reverts commit 25ffa03a26bc1578160df7d7a6bb650fc6753e9b.

Exit-9B committed:
Fix build with latest mob

Exit-9B committed:
Update workflow

MK-HATERS commented:
ty for the updates!!!!

MK-HATERS commented:
plugin has only Starfield.esm as master.
Any internal ID of FE000xxx, FD00xxxx, or 01xxxxxx or higher will all be treated as new records.

If there is for example:

plugin has Starfield.esm and a small flagged master "SmallMaster.esm" then
Any internal ID of FE001xxx, FD00xxxx, or 01xxxxxx or higher will all be treated as new records.

Note that the FE index for new records incremented because of the small master in the master list.

MK-HATERS commented:
wbRecord(TES4, 'Main File Header',
    wbFlags(wbRecordFlagsFlags, wbFlagsList([
      {0x001}  0, 'Master',
      {0x010}  4, 'Optimized',
      {0x080}  7, 'Localized',
      {0x100}  8, 'Small',
      {0x200}  9, 'Update',
      {0x400} 10, 'Medium',
      {0x800} 11, 'Blueprint'
    ], False), True), [

Exit-9B committed:
Load blueprint plugins last

Exit-9B committed:
Fix issue with plugin reordering algorithm

Silarn committed:
Port upstream changes & OR support

Silarn committed:
Add support for modDataDirectory

Silarn committed:
Upstream fixes
- Utilize vcpkg build
- Use native python
- Fix bsatk include
- Update SafeWriteFile
- Use strerror_s

Exit-9B committed:
Merge pull request #19 from Silarn/update-mob

Port upstream changes & OR support
